app/assets/scripts/map.js

```diff
@@ -157,6 +157,40 @@ $(document).ready(function() {
     });
 });
 
+function getNextDate(startDate) {
+    date = startDate.getDate();
+    month = startDate.getMonth();
+    year = startDate.getFullYear();
+    MAX_MONTH = 11;
+    monthDays = {
+        0: 31,
+        1: 28,
+        2: 31,
+        3: 30,
+        4: 31,
+        5: 30,
+        6: 31,
+        7: 31,
+        8: 30,
+        9: 31,
+        10: 30,
+        11: 31
+    }
+    if ((month == 1) && ((year % 400 == 0) || ((year % 100 != 0) && (year % 4 == 0)))) {
```

Comment:
Yeah, I think he accidentally pushed this code here. Please review this code at #83